charts/minibroker/templates/deployment.yaml

```diff
@@ -49,13 +49,8 @@ spec:
         - -logtostderr
         ports:
         - containerPort: 8080
-      - name: tiller
-        image: "gcr.io/kubernetes-helm/tiller:v2.13.1"
-        imagePullPolicy: IfNotPresent
         env:
-        - name: TILLER_NAMESPACE
-          value: {{ .Release.Namespace }}
-        - name: TILLER_HISTORY_MAX
-          value: "1"
-        command: ["/tiller"]
-        args: ["--listen=localhost:44134"]
```

Comment:
Hmmm this shouldn't be here. This will be part of the Helm 3 PR.

Reply:
solved!